components/ui/PageHeader.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <header class="page-header">
+    <div class="page-header__container">
+      <h1 class="page-header__title">
+        <slot />
+      </h1>
+    </div>
+  </header>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {}
+</script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
+@import '~carbon-components/scss/globals/scss/typography';
+@include carbon--theme($carbon--theme--white);
```

Comment:
If I don't recall wrong, I think this is the first time I see adding the `carbon--theme` mixing in a component. Why are we doing this?

Reply:
Per offline conversations, it's a passing experiment. I will remove it